skills/skill-writer/reference.md

```
@@ -0,0 +1,52 @@
+
+2. **Check file location**:
+   ```bash
+   # Claude Code
+   ls ~/.claude/skills/skill-name/SKILL.md
+   ls .claude/skills/skill-name/SKILL.md
+
+   # Other agents: check your agent's skill directory
+   ```
+
+3. **Validate YAML**:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)